x-pack/test/functional/services/ml/data_visualizer_index_based.ts

```diff
@@ -37,12 +37,88 @@ export function MachineLearningDataVisualizerIndexBasedProvider(
       await this.assertTotalDocumentCount(expectedFormattedTotalDocCount);
     },
 
-    async assertFieldsPanelsExist(expectedPanelCount: number) {
-      const allPanels = await testSubjects.findAll('~mlDataVisualizerFieldsPanel');
-      expect(allPanels).to.have.length(
-        expectedPanelCount,
-        `Expected field panels count to be '${expectedPanelCount}' (got '${allPanels.length}')`
-      );
+    async assertDataVisualizerTotalDocCountHeaderExist() {
```

Comment:
Nit: Since we're already in the `data_visualizer_index_based` service, I think we don't necessarily need to repeat `DataVisualizer` in the method names.